### PR TITLE
beardofedu tampa file

### LINKED ISSUE #4: beardofedu's checklist
# My checklist 

- [x] Create branch
- [ ] Work Locally
- [ ] Create commit(s)
- [ ] Create Pull Request
- [ ] Collaborate
- [ ] Merge!

### PR DESCRIPTION
**Pull request by githubteacher**
Creating a list of places to eat in Tampa, Fl.

Closes #4

### CODE REVIEW

**Review by rheadmar — COMMENTED** [review]
Is there anything good nearby to buy food and bring to the 7th Sun Brewery?

File: beardofedu-tampa.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Places to Eat
+
+- Dunedin Brewery
+- 7th Sun Brewery (bring your own food ....)
+- Demitiri's on the Water
```
Comment (rheadmar) [inline beardofedu-tampa.md]: What is good here?

File: beardofedu-tampa.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Places to Eat
+
+- Dunedin Brewery
+- 7th Sun Brewery (bring your own food ....)
```
Comment (rheadmar) [inline beardofedu-tampa.md]: Is there good food nearby?

**Review by rheadmar — COMMENTED** [review]
Not a bad list, do you hold training sessions at any of these locations?

**Comment by githubteacher** [comment]
> Not a bad list, do you hold training sessions at any of these locations? 

@rheadmar No, but they do have a drum circle every week .... 🥁

**Pull request merged**